kymatio/scattering2d/tests/test_utils_scattering2d.py

```diff
@@ -0,0 +1,13 @@
+import pytest
```

Comment:
Remove. I left this file in b/c I thought it was testing utility functions, not numpy frontend.

Reply:
Why should we remove?(isn't it a test file?)

Reply:
Actually, nevermind. Thought it was exporting a NP frontend that wasn't there.